src/spec_tools/transform.cljc

```diff
@@ -71,6 +71,13 @@
       :else x)
     x))
 
+(defn string->decimal [_ x]
```

Comment:
whole function can be wrapped in conditional

Reply:
clj conditional was added in the new commit! o/